packages/components/src/index.ts

```diff
@@ -129,15 +129,21 @@ export { default as __experimentalNavigationGroup } from './navigation/group';
 export { default as __experimentalNavigationItem } from './navigation/item';
 export { default as __experimentalNavigationMenu } from './navigation/menu';
 export {
+	/** @deprecated Import `Navigator` instead. */
 	NavigatorProvider as __experimentalNavigatorProvider,
+	/** @deprecated Import `Navigator` instead. */
 	NavigatorScreen as __experimentalNavigatorScreen,
+	/** @deprecated Import `Navigator` instead. */
```

Comment:
Will these comments be more useful if we do it like like this:

```suggestion
	/** @deprecated Import `Navigator` and use `Navigator.Button` instead. */
```

Reply:
Good point, I'll update those

Reply:
https://github.com/WordPress/gutenberg/pull/65802/commits/bd9ea197418d6d5e01853a4dbffa57060e0775d4